pkgs/libevent/build.textproto

```diff
@@ -0,0 +1,7 @@
+source: "https://github.com/libevent/libevent/releases/download/release-2.1.11-stable/libevent-2.1.11-stable.tar.gz"
+hash: "a65bac6202ea8c5609fd5c7e480e6d25de467ea1917c08290c521752f147283d"
+version: "2.1.11"
```

Comment:
Please make this 2.1.11-1

Reply:
It took me a while to realize why they were this way, this was the very first thing I've done and I didn't know about the scaffold command yet